src/data/faq_de.json

```
@@ -91,6 +91,17 @@
                 "title": "News",
                 "id": "news",
                 "accordion": [
+                    {
+                        "title": "Erklärung zur Sicherheitslücke in Log4J",
+                        "anchor": "log4j",
+                        "active": true,
+                        "textblock": [
+                            "Log4j ist ein weit verbreitetes Java-basiertes Werkzeug zur Protokollierung von Fehlermeldungen in Anwendungen.",
+                            "Die Log4j-Bibliothek wird in einem breiten Spektrum von Unternehmenssoftware und in Diensten vieler großer Anbieter eingesetzt. Kürzlich wurde eine Sicherheitslücke im Zusammenhang mit der Log4j-Bibliothek entdeckt, die die Ausführung von Remote-Code ermöglicht.",
+                            "Als Reaktion auf diese jüngste Entdeckung und um mögliche Auswirkungen zu minimieren, haben sich die Projektteams von SAP und TSI den weltweiten Bemühungen angeschlossen und Maßnahmen ergriffen. Dazu gehört unter anderem ein Versionsupdate auf 2.16.0 ",

```

Suggest to explicitly state this is about server software, not the app software.

```suggestion
                            "Als Reaktion auf diese jüngste Entdeckung und um mögliche Auswirkungen zu minimieren, haben sich die Projektteams von SAP und TSI den weltweiten Bemühungen angeschlossen und Maßnahmen ergriffen. Dazu gehört unter anderem ein Server-Versionsupdate auf 2.16.0 ",
```

---

Mhh, this sounds like the server was updated, but in fact, only log4j was updated. But this is just a minor detail.